autoware.repos

```diff
@@ -107,7 +107,7 @@ repositories:
   sensor_kit/external/awsim_sensor_kit_launch: # TODO: Integrate into sample_sensor_kit_launch
     type: git
     url: https://github.com/tier4/awsim_sensor_kit_launch.git
-    version: 0.39.0
+    version: v0.40.0
```

Comment:
`v` prefix version is not acceptable.

Reply:
Thank you for the correction.

Reply:
@youtalk Thank you for detecting the bug. Let me investigate!